src/index.ts

```diff
@@ -1,11 +1,13 @@
+import { Readable } from 'node:stream';
+import { parse } from 'node:url';
 import type { WithRequired } from '@apollo/utils.withrequired';
+import type { HTTPGraphQLResponseBody } from '@apollo/server/dist/esm/externalTypes/http';
```

Comment:
We shouldn't import from `/dist`. This type isn't publicly exported (though it could be).

For now, we could access the type via `HTTPGraphQLResponse['body']`.

Though I'd argue that where we're using it, TypeScript is actually correct in that it's a `never`. Whatever we have in that `else` conditional is _not_ an `HTTPGraphQLResponseBody` and we should probably handle it as an `unknown` and just throw a generic programming error that says the body was an unexpected type.

Reply:
Thanks, this was auto-suggested by VS Code and I didn't look at it too hard. This symbol is not exported by Apollo so I guess we'll just have to use `any` due to the enum type narrowing in the bottom `else`.